cicd/1-setup/cicd-dependencies.template.yml

```diff
@@ -27,7 +27,6 @@ Resources:
                 - codepipeline.amazonaws.com
         Version: '2012-10-17'
       Path: /service-role/
-      PermissionsBoundary: !ImportValue IAM-DevPermissions
```

Comment:
This is a tentative removal. I don't think we need it here. This stack is being deployed by an Admin anyway. I might be overlooking something though.